String/ValidateCreditCard.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Add references and a little explanation of how this works.

Reply:
Also add a little description inside the comments, apart from the reference.

Reply:
Uh, that is not what I meant. See [this](https://github.com/TheAlgorithms/Javascript/blob/master/Maths/FermatPrimalityTest.js) algorithm for example. They have the external reference, but also provide a short description of how the algorithm works.

Reply:
The code should be self explanatory. The reference and the single line comments should do it.